src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsApiWriter.java

```diff
@@ -166,6 +170,16 @@ private void sendRequest(JsonArray content, URL url) throws IOException {
         con.disconnect();
     }
 
+    private void setRequestProperties(HttpURLConnection con) {
+        con.setRequestProperty("Content-Type", "application/json");
+        con.setRequestProperty("Content-Encoding", "gzip");
+        con.setRequestProperty("DD-API-KEY", config.ddApiKey);
+
+        Package p = getClass().getPackage();
+        con.setRequestProperty("DD-EVP-ORIGIN", p.getName());
```

Comment:
I was thinking `datadog-kafka-connect-logs` would make more sense for this header (and may more consistent with the value for other logs source), is there a reason you went for the java package name instead?

Reply:
I have set `DD-EVP-ORIGIN` to the project name instead of the package name.